client/src/panels/panel_declarations/results/results_intro.js

```diff
@@ -0,0 +1,123 @@
+import './results.scss';
+
+import classNames from 'classnames';
+import { Fragment } from 'react';
+import {
+  declare_panel,
+  declarative_charts,
+  InfographicPanel,
+  businessConstants,
+  get_source_links,
+  create_text_maker_component,
+} from "../shared.js";
+import text from './results_intro_text.yaml';
+import { get_static_url } from '../../../request_utils.js';
+import { 
+  ResultCounts,
+  GranularResultCounts,
+  row_to_drr_status_counts,
+  get_result_doc_keys,
+  result_docs,
+} from './results_common.js';
+
+const { text_maker, TM } = create_text_maker_component(text);
+
+
+const latest_drr_doc_key = _.last( get_result_doc_keys("drr") );
+const latest_dp_doc_key = _.last( get_result_doc_keys("dp") );
+
+const ResultsIntroPanel = ({subject, summary_counts, doc_urls}) => {
+  const summary_text_args = {subject, ...summary_counts};
+  
+  return (
+    <div className="frow middle-xs">
+      <div className="fcol-md-7 medium_panel_text">
+        <TM k="results_intro_text" />
+      </div>
+      {!window.is_a11y_mode &&
+        <div className="fcol-md-5">
+          <div
+            style={{
+              padding: "20px",
+            }}
+          >
+            <img
+              src={get_static_url(`png/result-taxonomy-${window.lang}.png`)} 
+              style={{
+                width: "100%",
+                maxHeight: "500px",
+              }}
+            />
+          </div>
+        </div>
+      }
+      <div className="fcol-md-12 medium_panel_text">
+        <TM k="dp_summary_text" args={summary_text_args} />
+        <TM k="drr_summary_text" args={summary_text_args} />
+        <TM k="reports_links_text" args={doc_urls} />
+      </div>
+    </div>
+  );
+};
+
+export const declare_results_intro_panel = () => declare_panel({
+  panel_key: "results_intro",
+  levels: ["gov", "dept"],
+  panel_config_func: (level, panel_key) => ({
+    requires_result_counts: level === "dept",
+    require_granular_result_counts: level !== "dept",
+    footnotes: ["RESULTS_COUNTS", "RESULTS"],
+    source: (subject) => get_source_links(["DRR"]),
+    calculate: (subject) => {
+      const verbose_counts = (() => {
+        switch (level){
+          case 'dept':
+            return ResultCounts.get_dept_counts(subject.id);
+          case 'gov':
+            return ResultCounts.get_gov_counts();
+        }
+      })();
+
+      if(verbose_counts[`${latest_drr_doc_key}_total`] < 1){
+        return false;
```

Comment:
Hmm, it would take complicating the text a bit (well, adding some conditionals to the yaml, which I know you don't like), but I think it would be good for this panel to display as long as the subject has EITHER the latest DRR or the latest DP.

I don't know any cases of it off the top of my head, but I think it would read right now in the case where a department has a DP but no DRR, as the panel will display but the DRR text will be full of 0's.

Reply:
Oh yeah good point. I think the DP and DRR text are separate keys so the conditional can be in the JS 😜

Reply:
Aha, that works too!